src/test/java/org/reaktivity/nukleus/http_cache/internal/streams/proxy/EdgeArchProxyIT.java

```diff
@@ -320,4 +321,55 @@ public void shouldMaintainPollingForMultipleAuthScopes() throws Exception
         k3po.finish();
         counters.assertExpectedCacheEntries(2, 0, 2);
     }
+
+
+    @Test
+    @Specification({
+        "${route}/proxy/controller",
+        "${streams}/no.authorization.sends.cache.control.private/accept/client",
+        "${streams}/no.authorization.sends.cache.control.private/connect/server",
+    })
+    public void noAuthorizationSendsCacheControlPrivate() throws Exception
+    {
+        k3po.finish();
+        //counters.assertExpectedCacheEntries(2, 0, 2);
```

Comment:
These commented out sections don't look good.  Do we know what the cause of that is